packages/gatsby/content/getting-started/install.md

```diff
@@ -26,6 +26,24 @@ Corepack isn't included with Node.js in versions before the 16.10; to address th
 npm i -g corepack
 ```
 
+## Ensuring Corepack's global Yarn is current
+
+Corepack won't always have the latest Yarn release. You can make sure it does with [`corepack prepare`](https://nodejs.org/api/corepack.html#upgrading-the-global-versions).
+
+### Node.js >=16.17 or >=18.6
```

Comment:
```suggestion
### Node.js ^16.17 or >=18.6
```

Reply:
Just looked through `*.md` files in this repo (without the changes in this PR):
`(?<!\[)\^\d+\b`: 22 results, all code sample or command examples.
`>=\d+\b`: 4 results (2 in CHANGELOG.md): this seems to be current norm for non-code-sample non-command-example version range references.

Two options:

A) Stay with old norm, modify this PR so that `packages/gatsby/content/getting-started/install.md` uses `>=` instead of `^`.

B) Modify [/packages/gatsby/content/features/protocols.md](https://github.com/yarnpkg/berry/blob/master/packages/gatsby/content/features/protocols.md) & [packages/gatsby/content/getting-started/editor-sdks.md](https://github.com/yarnpkg/berry/blob/master/packages/gatsby/content/getting-started/editor-sdks.md) to use `^` instead of `>=`.

Reply:
@merceyz, sorry to bug ya, but the way I resolved your request left this PR looking unsettled. I believe it is once again ready for (re)review. Thanks!

Reply:
The reason for my suggestion here is that `>=16.17` already includes `>=18.6`